platform-core-module/src/main/java/com/softicar/platform/core/module/locale/AGLocalizationPreset.sqml

```diff
@@ -0,0 +1,14 @@
+PACKAGE com.softicar.platform.core.module.locale
+
+IMPORT com.softicar.platform.core.module.language.AGCoreLanguage
+
+TABLE AGLocalizationPreset 'Core.LocalizationPreset' {
+	PK Integer id = SERIAL
+	String name [MAXLENGTH=255]
+	
+	AGCoreLanguage language
+	String decimalSeparator = '.' [MAXLENGTH=255]
```

Comment:
I don't think we should define a default here.

Reply:
well, done